src/focus_manager/txm_focus_manager.js

```diff
@@ -436,8 +436,16 @@ export class TXMFocusManager {
             }
 
         } else if (this.isInContent(focus)) {
-            // Focus is in the content area.
-            newFocus = this.findNextFocus(focus, action, this._contentFocusables);
+            // Focus is in the content area. If we are possibly moving to a footer, try allow a move to it via the
+            // the "natural" focus lane rules first.
+            var possibleFocusables = this._contentFocusables;
+            if (action == inputActions.moveUp && this._lastTopFocus) {
+                possibleFocusables = possibleFocusables.concat(this._lastTopFocus);
+            } else if (action == inputActions.moveDown && this._lastBottomFocus) {
+                possibleFocusables = possibleFocusables.concat(this._lastBottomFocus);
+            }
+
+            newFocus = this.findNextFocus(focus, action, possibleFocusables);
             if (!newFocus) {
```

Comment:
Do we still need the handling here if you added the logic above?  I'm trying to picture a case you would have no focus, but you press down, it would explicitly set focus to it.  When in theory, the generalized focus rules should always go to it now at this point.

Reply:
I was considering it, but decided to be conservative and leave it alone, since I was recalling the content/footer overlap case you ran into, and this is the code that lets one escape from that.

Reply:
Which case is that?   Still sounds like findNextFocus should take care of that scenario once you add it to the focusables.